.tekton/aws-load-balancer-operator-container-aws-lb-optr-1-3-rhel-9-push.yaml

```diff
@@ -201,6 +201,8 @@ spec:
       - name: BUILD_ARGS
         value:
         - $(params.build-args[*])
+        - 'FULL_COMMIT=$(tasks.clone-repository.results.short-commit)'
+        - 'GIT_URL=$(tasks.clone-repository.results.url)'
```

Comment:
Where is the corresponding update in the operator's Containerfile? [Related comment](https://github.com/openshift/aws-load-balancer-operator/pull/215#issuecomment-3365397810).